### PR TITLE
new test: scollElement-002.html

### PR DESCRIPTION
**Pull request by tigercosmos**
Add a new test on `element.scrollTop` and `element.scrollLeft` for:
- `margin` and `padding`
- `overflow: hidden` and `overflow: scroll`

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/20995)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/305989257)

File: lint.whitelist
Comment (jdm) [inline lint.whitelist]: Instead of adding an exception, this test should contain links to the relevant specification like https://github.com/tigercosmos/web-platform-tests/blob/3f589840950e74cb4918709cdcb6b6e59c57565a/css/cssom-view/cssom-getBoundingClientRect-002.html#L7-L8 .

**Review by jdm — CHANGES_REQUESTED** [review]
I have not actually reviewed the test content (and don't really feel qualified to).

**Comment by tigercosmos** [comment]
@jdm  Your are right. I fill up links.
Could you give some advise about the test?

**Comment by foolip** [comment]
@tobie, do you know why @wpt-pr-bot added the infra label to this?

**Comment by tobie** [comment]
@foolip because it originally touched `/lint.whitelist` before being rebased.

**Comment by foolip** [comment]
@tobie, where's the code for this? I can't find "lint" in the files you linked to in https://github.com/w3c/web-platform-tests/issues/8286#issuecomment-345093309

(I think that no label at all would be appropriate when touching just lint.whitelist.)

**Comment by tobie** [comment]
@foolip: https://github.com/tobie/ganesh/blob/master/lib/metadata/raw-labels.js#L65-L67

**Comment by tigercosmos** [comment]
@plinss  @jgraham could you take a look?

**Comment by tigercosmos** [comment]
Hello~ Anyone?

File: css/cssom-view/elementScroll-002.html
Comment (jdm) [inline css/cssom-view/elementScroll-002.html]: I don't see a reason to wait for onload here. Additionally, if we remove the setup call, I think the test should complete without any problems.

**Review by jdm — CHANGES_REQUESTED** [review]

**Review by jdm — APPROVED** [review]

**Comment by tigercosmos** [comment]
@jdm Should I ping someone after you have approved?

**Comment by jdm** [comment]
The TravisCI failure was fixed by another PR that merged 6 hours ago, so this can merge.

**Comment by jdm** [comment]
Ugh, I can't merge this until I get access to a non-mobike device. If you rebase the change against master that might help.

**Comment by tigercosmos** [comment]
@jdm  It's all green now. Does this mean it can be merged?

**Comment by tigercosmos** [comment]
Thanks!

**Pull request merged**
